src/net/atlas/db.rs

```diff
@@ -396,10 +415,11 @@ impl AtlasDB {
         is_available: bool,
     ) -> Result<(), db_error> {
         let hex_content_hash = to_hex(&attachment.content_hash.0[..]);
+        let hex_tx_id = to_hex(&attachment.tx_id.0[..]);
```

Comment:
This might be a little less clunky of you use `&attachment.tx_id.to_hex()` (the `.to_hex()` method comes with the `impl_byte_array_newtype!` macro, which is invoked for `Txid` and `Hash160`).

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2453/commits/672d70326ca6c4ff6cf49dfbb6c69783d6696ca2